packages/embark-whisper/src/index.js

```diff
@@ -24,6 +24,7 @@ class Whisper {
     this.embark = embark;
     this.web3Ready = false;
     this.webSocketsChannels = {};
+    this.modulesPath = dappPath(embark.config.embarkConfig.generationDir + "/modules");
```

Comment:
/modules should be put in a constant and used everywhere from the symlink generation to the uses in modules

Reply:
```suggestion
    this.modulesPath = dappPath(embark.config.embarkConfig.generationDir, "modules");
```

Also should make use of the path-joining that `dappPath()` does because doing `+ "/"` is problematic on Windows.